index.html

```diff
@@ -9,5 +9,24 @@
 </head>
 <body>
     <h1>Hangman</h1>
+
+    <div>      
+        <img id="gallows" src="./assets/gallow.png" alt="gallow">
+        <img id="head" src="./assets/head.png" alt="head">
+        <img id="body" src="./assets/body.png" alt="body">
+        <img id="left-arm" src="./assets/left-arm.png" alt="left-arm">
+        <img id="right-arm" src="./assets/right-arm.png" alt="right-arm">
+        <img id="left-leg" src="./assets/left-leg.png" alt="left-leg">
+        <img id="right-leg" src="./assets/right-leg.png" alt="right-leg">
+    </div>
+
+    <section id = 'empty-word' class="word-being-guessed">
```

Comment:
Eep, looks like you have a couple of extra spaces `id = 'empty-word'` should be `id="empty-word"`